app/org/sagebionetworks/bridge/models/accounts/StudyParticipant.java

```diff
@@ -273,6 +283,9 @@ public Builder copyFieldsOf(StudyParticipant participant, Set<String> fieldNames
             if (fieldNames.contains("timeZone")) {
                 this.timeZone = participant.getTimeZone();    
             }
+            if (fieldNames.contains("clientData")) {
+                this.clientData = participant.getClientData();
+            }
```

Comment:
Is it worth adding a test for copyFieldsOf()?

Reply:
Added tests for each field in copyFieldsOf() to verify that only the indicated field is copied.